test/base/ByteBufferTest.ooc

```diff
@@ -1,3 +1,48 @@
 use ooc-base
+use ooc-unit
 
-"ByteBufferTest [TODO: Not implemented as a fixture!]" printfln()
+ByteBufferTest: class extends Fixture {
+	init: func {
+		super("ByteBuffer")
+		this add("set data", func {
+			buffer := ByteBuffer new(1024)
+			expect(buffer size, is equal to(1024))
+			for (i in 0 .. 1024 / 8)
+				buffer pointer[i] = i
+			for (i in 0 .. 1024 / 8)
+				expect(buffer pointer[i] as Int, is equal to(i))
+			buffer free()
```

Comment:
Always do `referenceCount decrease()` on things that have a refcounter, don't free them like this.